Classes/ViewHelpers/Be/ConfigurationViewHelper.php

```diff
@@ -33,7 +33,7 @@ public function render()
     {
         $this->pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
 
-        $baseUrl = '../' . ExtensionManagementUtility::siteRelPath('extension_builder');
+        $baseUrl = ExtensionManagementUtility::extPath('extension_builder');
         $this->pageRenderer->disableCompressJavascript();
         $this->pageRenderer->addJsFile($baseUrl . 'Resources/Public/jsDomainModeling/jquery.min.js');
```

Comment:
This should better use the "EXT:extension_builder/Resources..." path spec

Reply:
In fact extPath is not working here. Has to be surrounded with
/TYPO3\CMS\Core\Utility\PathUtility::getAbsoluteWebPath()

Reply:
$baseUrl = PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('extension_builder'));